src/app/header/header.component.html

```diff
@@ -15,6 +15,9 @@ <h1>BookMyShow</h1>
         <li class="nav-item" routerLinkActive="active">
           <a class="nav-link" href="#" routerLink="/people">People</a>
         </li>
+        <li class="nav-item" routerLinkActive="active">
+          <a class="nav-link" href="#" routerLink="/people">Discover</a>
```

Comment:
@truptighatol duplicate router link `/people`, please fix this

Reply:
it should be `/discover` instead `/people`

Reply:
ok